Modules/IO/NIFTI/include/itkNiftiImageIO.h

```diff
@@ -25,6 +25,10 @@
 #include <memory>
 #include "itkImageIOBase.h"
 
+
+// Forward declaration of the nifti_image structure
+struct nifti_image_struct;
```

Comment:
Could the existing `nifti_image` typedef be used here?

Reply:
We cannot include the Nifti header here. The original nifti_image type was a typedef of an anonymous struct.

I do not know of a better solution.